### PR TITLE
Fix entity mapping for GitHub detections

### PR DESCRIPTION
**Pull request by itay6588**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Detections/SigninLogs/Brute Force Attack against GitHub Account.yaml [hunk 1]
```diff
@@ -30,14 +30,14 @@ query: |
...
+  | extend LearningThreshold = max_of(AvgOfFailedLoginsInLearning + StdOfFailedLoginsInLearning * NumberOfStds, MinThreshold)  
   | join kind=innerunique ( 
     GitHubFailedSSOLogins 
     | where TimeGenerated between (ago(StartTime) .. ago(EndRunTime)) 
     | summarize FailedLoginsCountInRunTime = count() by User = Identity 
-  ) on User 
+  ) on UserPrincipalName 
   | where FailedLoginsCountInRunTime > LearningThreshold
-  //| extend AccountCustomEntity = UserPrincipalName, IPCustomEntity = IPAddress
+  | extend AccountCustomEntity = UserPrincipalName, IPCustomEntity = set_IPAddress
```
Comment (shainw) [inline Detections/SigninLogs/Brute Force Attack against GitHub Account.yaml:42]: We cannot present a dynamic value in the custom entities.  Normally, we will use mv-expand on the set if we want to include.  So, you can do this or drop IPCustomEntity.

File: Detections/SigninLogs/GitHub Activities from Infrequent Country.yaml [hunk 2]
```diff
@@ -26,11 +26,11 @@ query: |
...
-  | summarize makeset(Country) by Actor
+  | summarize makeset(Country), makeset(IPaddress) by Actor
   | join kind=innerunique (
     GitHubCountryCodeLogs
     | where TimeGenerated between (ago(StartTime) .. ago(EndRunTime))
     | distinct Country, Actor 
   ) on Actor 
   | where set_Country !contains Country
-  //| extend AccountCustomEntity = Actor, IPCustomEntity = IPaddress
+  | extend AccountCustomEntity = Actor, IPCustomEntity = set_IPaddress
```
Comment (shainw) [inline Detections/SigninLogs/GitHub Activities from Infrequent Country.yaml:36]: Same as above.

File: Detections/SigninLogs/Sign-in Burst from Multiple Locations.yaml [hunk 3]
```diff
@@ -22,8 +22,8 @@ query: |
   | where TimeGenerated > ago(RunTime)
   | where AppDisplayName == "GitHub.com"
   | where ResultType == 0
-  | summarize CountOfLocations = dcount(Location), Locations = make_set(Location) by User = Identity
+  | summarize CountOfLocations = dcount(Location), Locations = make_set(Location), IPs=make_set(IPAddress) by UserPrincipalName
   | where CountOfLocations > 1
-  //| extend AccountCustomEntity = UserPrincipalName, IPCustomEntity = IPAddress
+  | extend AccountCustomEntity = UserPrincipalName, IPCustomEntity = IPs
```
Comment (shainw) [inline Detections/SigninLogs/Sign-in Burst from Multiple Locations.yaml:27]: Same as above.

**Comment by itay6588** [comment]
we'll be fixed in another PR regarding UEBA

**Pull request closed without merging**